bdd100k/eval/mot.py

```diff
@@ -32,24 +41,30 @@
 IGNORE_CLASSES = ["trailer", "other person", "other vehicle"]
 
 
-def parse_objects(objects: List[DictAny]) -> List[np.ndarray]:
+def parse_objects(objects: List[Label]) -> List[np.ndarray]:
     """Parse objects under Scalable formats."""
     bboxes, labels, ids, ignore_bboxes = [], [], [], []
     for obj in objects:
+        box2d = obj.box2d
```

Comment:
in `Label`, the underscore in `box_2d` is necessary. You can check the unit tests in scalabel.